### DISCUSSION TITLE
Embed dashboard editor UI

### DISCUSSION THREAD

**Discussion by 7840vz** [perses/perses#2310] [Design Discussions]
Hi Perses team. Could help me to figure out, is "dashboard builder" ui (app folder I think in the code), is it something that can be easily embedded within my own app, without hard forking Perses? Just like I can do now with dashboards themselves with typescript library? If not, is this something that can come in future or this case is out of scope? 

Our use case is that we want to allow creating simple dashboards within our own webapp.
![image](https://github.com/user-attachments/assets/b815c3d6-345c-48d1-b5ba-ccb906d30b3d)

Thanks.

**Comment by Nexucis** [comment]
sorry that nobody answer you. But what you should import is the package `@perses-dev/dashboards` that provides you the dashboard view.

The app is not something we would like to export. Of course we can discuss if you there are things in the app you would like to see exposed. But generally speaking what we have developed in the app is an example of an implementation.
